R/issues.R

```diff
@@ -116,6 +116,51 @@ get_issue_assignees <- function(org, repo, .api_url = api_url(), pages = NULL){
 	return(assignees %>% select(issue, everything()))
 }
 
+#' Gets a data frame of the assignees and participants for issues
+#' @inheritParams ghpm
+#' @return A data frame containing issue | assignedTo of each issue. Returns an empty dataframe if none are found.
+#' @importFrom purrr reduce map_df keep
+#' @importFrom tibble tibble add_row
+#' @importFrom dplyr mutate select everything bind_rows
+#' @export
+get_issues_assignees_participants <- function(org, repo, .api_url = api_url(), pages = NULL){
+	data <- get_query_results(
+		gql_file="issues/issues_assignees_participants.graphql",
+		param_list = c("repository", "issues"),
+		pages = pages,
+		org = org,
+		repo = repo,
+		.api_url = .api_url
+	)
+
+	data <- keep(data, ~ (.x$assignees$total_count > 0 || .x$participants$total_count > 0))
```

Comment:
This doesn't seem to filter out all the data where the total_count is greater than 0. I'm getting an error on line 149 (`assigndat$total_count <- x$assignees$total_count`):

```
`$<-.data.frame`(`*tmp*`, "total_count", value = 0L) : replacement has 1 row, data has 0
```

Reply:
nvm this has been resolved with the latest tibble release